spec/requests/api/v1/game_request_spec.rb

```diff
@@ -0,0 +1,20 @@
+require 'rails_helper'
+
+RSpec.describe 'Game endpoint', type: :request do
```

Comment:
This seems like a duplicate of the other tests.

Reply:
request specs are more like integration tests which cover everything being tested in the controller specs